woocommerce/woocommerce
# Deprecate E2E setup and onboarding tests

This PR accidentally has the commit from #31239. We'll hold this until it has been merged.

### Changes proposed in this Pull Request:

This PR adds deprecation notices to the setup and onboarding tests in e2e-core-tests in favor of the tests in `@woocommerce/admin-e2e-tests`.

Closes #30866 .

### How to test the changes in this Pull Request:

1. Create `plugins/woocommerce/tests/e2e/specs/test-30866.js` with the following
```
const { runSetupOnboardingTests } = require( '@woocommerce/e2e-core-tests' );

runSetupOnboardingTests();
```
2. Run `cd plugins/woocommerce`
3. Run `pnpx wc-e2e tests/e2e/specs/test-30866.js`
4. This should produce a deprecation message for each of the four tests
